lib/client.js

```diff
@@ -63,6 +63,10 @@ function SMTPClient(port, host, options) {
     this.options.maxConnections = this.options.maxConnections || 5;
     this.options.enableDotEscaping = this.options.enableDotEscaping || false;
 
+    if (this.options.instanceId) {
+        this.options.instanceId = process.pid + '-' + this.options.instanceId
```

Comment:
What will `instanceId` be here?

Reply:
instanceId is an optional string or number passed in on construction that is used to differentiate between log statements from different connections. Its used purely informationally.